apis/basic/composition.yaml

```diff
@@ -6,528 +6,272 @@ metadata:
-                      type: Format
-              - type: ToCompositeFieldPath
-                fromFieldPath: metadata.annotations[crossplane.io/external-name]
-                toFieldPath: status.subnetIds[1]
-              - type: ToCompositeFieldPath
-                fromFieldPath: metadata.annotations[crossplane.io/external-name]
-                toFieldPath: status.publicSubnetIds[1]
+            ---
+            apiVersion: ec2.aws.upbound.io/v1beta1
+            kind: InternetGateway
+            metadata:
+              labels:
+                {{- include "networkId" . | nindent 4 }}
+              annotations:
+                {{ setResourceNameAnnotation ("igw") }}
+            spec:
+              forProvider:
+                vpcIdSelector:
+                  matchControllerRef: true
+                {{- include "region" . | nindent 2 }}
+              {{- include "providerConfigRef" . }}
+              {{- include "deletionPolicy" . }}
 
-          - name: subnetPrivateA
-            base:
-              apiVersion: ec2.aws.upbound.io/v1beta1
-              kind: Subnet
-              metadata:
-                labels:
-                  access: private
-              spec:
-                forProvider:
-                  cidrBlock: 192.168.128.0/18
-                  tags:
-                    kubernetes.io/role/internal-elb: "1"
-                  vpcIdSelector:
-                    matchControllerRef: true
-            patches:
-              - type: PatchSet
-                patchSetName: providerConfigRef
-              - type: PatchSet
-                patchSetName: deletionPolicy
-              - type: PatchSet
-                patchSetName: network-id
-              - type: PatchSet
-                patchSetName: region
-              - type: FromCompositeFieldPath
-                fromFieldPath: spec.parameters.region
-                toFieldPath: spec.forProvider.availabilityZone
-                transforms:
-                  - type: string
-                    string:
-                      fmt: '%sa'
-                      type: Format
-              - type: FromCompositeFieldPath
-                fromFieldPath: spec.parameters.region
-                toFieldPath: metadata.labels[zone]
-                transforms:
-                  - type: string
-                    string:
-                      fmt: '%sa'
-                      type: Format
-              - type: ToCompositeFieldPath
-                fromFieldPath: metadata.annotations[crossplane.io/external-name]
-                toFieldPath: status.subnetIds[2]
-              - type: ToCompositeFieldPath
-                fromFieldPath: metadata.annotations[crossplane.io/external-name]
-                toFieldPath: status.privateSubnetIds[0]
+            {{ $region := .observed.composite.resource.spec.parameters.region }}
+            {{ $deletionPolicy := .observed.composite.resource.spec.parameters.deletionPolicy | default "Delete" }}
```

Comment:
Why do we need to change the style of the `region` and `deletionPolicy` here compared to the above-used defined/include? Let's try to stick to one consistent style

Reply:
```suggestion
            {{ $deletionPolicy := $params.deletionPolicy | default "Delete" }}
```